platform/android/src/jni.hpp

```diff
@@ -103,6 +103,62 @@ extern jclass httpRequestClass;
 extern jmethodID httpRequestStartId;
 extern jmethodID httpRequestCancelId;
 
+// Offline declarations start
+
+extern jclass offlineManagerClass;
+extern jmethodID offlineManagerClassConstructorId;
```

Comment:
Removed in https://github.com/mapbox/mapbox-gl-native/commit/4e1101d967f999b5bb35c397721f9649416ba938.